app.js

```diff
@@ -5,27 +5,51 @@ const mongoose = require("mongoose");
 const cors = require("cors");
 require("dotenv").config();
```

Comment:
```suggestion
require("dotenv").config();
const cors = require("cors");
```
`require("dotenv").config()`가 환경 변수를 빨리 불러오게 하기 위해 대게 맨 첫줄에 실행한다고 하더라고요
이렇게 바꿔주시면 감사하겠습니다 💯